projects/plugins/crm/.eslintrc.js

```diff
@@ -7,5 +7,6 @@ module.exports = {
 				allowedTextDomain: 'zero-bs-crm',
 			},
 		],
+		'jsdoc/require-jsdoc': 'off',
```

Comment:
I disabled this because the rule is inconsistent in lamba functions vs function declarations, and there was a conversation in Slack recently regarding this.